packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -439,55 +423,62 @@ abstract class JavaScriptChannel {
 /// Receive various notifications and requests for [WebView].
 abstract class WebViewClient {
   /// Constructs a [WebViewClient].
-  WebViewClient({this.shouldOverrideUrlLoading = true});
+  WebViewClient({this.shouldOverrideUrlLoading = true}) {
+    if (!_flutterApisHaveBeenSetup) {
+      WebViewClientFlutterApi.setup(WebViewClientFlutterApiImpl());
+      _flutterApisHaveBeenSetup = true;
+    }
+  }
+
+  static bool _flutterApisHaveBeenSetup = false;
 
   /// User authentication failed on server.
-  static const int errorAuthentication = 0xfffffffc;
+  static const int errorAuthentication = -4;
```

Comment:
Are these Java constants? If these must match something else, the comments should explain what that something is (e.g., with a docs link)

Reply:
Done. I added the java links to each one